plugin/fuzz/source/dextool/plugin/fuzz/frontend/fuzz.d

```diff
@@ -13,6 +13,35 @@ import dextool.compilation_db;
 
 import xml_parse;
 
+//Search and get headers from specific file
+auto getHeaderFiles(CompileCommandDB compile_db, CompileCommand file) @safe {
+    import std.range : array;
+    import std.algorithm : uniq, filter, map, sort;
+    import std.file : dirEntries, SpanMode;
+    import std.conv : to;
+
+    string[] flags = parseFlag(file, CompileCommandFilter()).filter!(a => a[0] != '-').array;
```

Comment:
tip: change CompilerCommandFilter to defaultCompilerFilter.
why? clang and gcc do not support the same flags. The default implementation of filter, defaultCompilerFilter, covers the most common deviations. It is empirically derived through trial and error.